docs/modules/release-notes/pages/5-4-0.adoc

```diff
@@ -0,0 +1,335 @@
+* Fixed an issue where the mechanism to retrieve partitioning strategy on a client was ignoring to check the provided Hazelcast cluster properties.
+https://github.com/hazelcast/hazelcast/pull/25162[#25162]
+* Fixed an issue where `ClientConfigXmlGenerator` didn't have the support for `hazelcast-cloud` configuration.
+https://github.com/hazelcast/hazelcast/pull/25155[#25155]
+* Fixed an issue where the cache provider was not able to read the YAML configurations.
+https://github.com/hazelcast/hazelcast/pull/25137[#25137]
+* Fixed an issue where the `getDistributedObjects()` was returning inconsistent results when multiple members are simultaneously joining to the cluster.
+https://github.com/hazelcast/hazelcast/pull/25114[#25114]
+* Fixed an issue where zero-config compact serialization was not working on the objects that have a field of type `java.util.UUID`.
+https://github.com/hazelcast/hazelcast/pull/25073[#25073]
+* Fixed an issue where the retry mechanism for the communications between CP leader and followers was generating too many retries, due to incorrect backoff timeout reset behavior.
+https://github.com/hazelcast/hazelcast/pull/25055[#25055]
+* Fixed an issue where there was a difference between the elapsed clock time and elapsed total time when listening to migration events.
+https://github.com/hazelcast/hazelcast/pull/25028[#25028]
+* Fixed an issue where the transaction in the Kafka producer was not committed when a batch job is finished.
+https://github.com/hazelcast/hazelcast/pull/25024[#25024]
+* Fixed an issue where data events were being fired through WAN replication after a split-brain, even there were no changes in data.
+https://github.com/hazelcast/hazelcast/pull/24928[#24928]
+* Fixed an issue where the lite members were not reporting statistics for map operations.
+https://github.com/hazelcast/hazelcast/pull/24871[#24871]
+* Fixed an issue where the blacklisting was being ignored after a split-brain happens.
+https://github.com/hazelcast/hazelcast/pull/24830[#24830]
+* Fixed an issue where the Kinesis sink might loose data, when retrying on failures, during a terminal snapshot.
+https://github.com/hazelcast/hazelcast/pull/24779[#24779]
+* Fixed an issue where the member list was not updated after a cluster failover scenario.
+https://github.com/hazelcast/hazelcast/pull/24745[#24745]
+* Fixed an issue where the batches produced for write-behind queues were not having the expected size of entries.
+ https://github.com/hazelcast/hazelcast/issues/24763[#24763]
+* Fixed an issue where the fused Jet vertex was ignoring the configured local parallelism and using the default parallelism instead.
+https://github.com/hazelcast/hazelcast/issues/24683[#24683]
+* Fixed an issue where Hazelcast was sending empty map interceptor information to the members which are newly joined to the cluster; it was causing eager map initializations.
+https://github.com/hazelcast/hazelcast/pull/24639[#24639]
```

Comment:
need to add something for anesoruk's Kafka partitions initial offsets contribution: https://github.com/hazelcast/hazelcast/pull/21546

Reply:
@rebekah-lawrence , the related PR is still open, and I agree with https://github.com/hazelcast/hz-docs/pull/996/files#r1479912945

Reply:
https://github.com/hazelcast/hz-docs/pull/1001/commits/479bf5651dc132c047124c3fd8277573cca8be97